content/source/docs/enterprise/release/index.html.md

```diff
@@ -0,0 +1,25 @@
+| [v202107-1](./v202107-1.html) |  550              | [2.50.0](https://release-notes.replicated.com/release-notes/2.50.0/)   |[1.0.1](https://github.com/hashicorp/terraform/releases/tag/v1.0.1)     | [0.18.3](https://docs.hashicorp.com/sentinel/changelog#0-18-3-june-1-2021)     |
+| [v202106-1](./v202106-1.html) |  544              | [2.50.0](https://release-notes.replicated.com/release-notes/2.50.0/)   |[1.0.0](https://github.com/hashicorp/terraform/releases/tag/v1.0.0)     | [0.18.3](https://docs.hashicorp.com/sentinel/changelog#0-18-3-june-1-2021)     |
+| [v202105-1](./v202105-1.html) |  534              | [2.50.0](https://release-notes.replicated.com/release-notes/2.50.0/)   |[0.15.3](https://github.com/hashicorp/terraform/releases/tag/v0.15.3)   | [0.18.1](https://docs.hashicorp.com/sentinel/changelog#0-18-1-may-11-2021)     |
+| [v202104-1](./v202104-1.html) |  528              | [2.50.0](https://release-notes.replicated.com/release-notes/2.50.0/)   |[0.15.0](https://github.com/hashicorp/terraform/releases/tag/v0.15.0)   | [0.18.0](https://docs.hashicorp.com/sentinel/changelog#0-18-0-march-25-2021)   |
+| [v202103-3](./v202103-3.html) |  523              | [2.29.0](https://release-notes.replicated.com/release-notes/2.29.0/)   |[0.14.7](https://github.com/hashicorp/terraform/releases/tag/v0.14.7)   | [0.17.4](https://docs.hashicorp.com/sentinel/changelog#0-17-4-february-2-2021) |
+| [v202103-2](./v202103-2.html) |  520              | [2.29.0](https://release-notes.replicated.com/release-notes/2.29.0/)   |[0.14.7](https://github.com/hashicorp/terraform/releases/tag/v0.14.7)   | [0.17.4](https://docs.hashicorp.com/sentinel/changelog#0-17-4-february-2-2021) |
+| [v202102-2](./v202102-2.html) |  509              | [2.29.0](https://release-notes.replicated.com/release-notes/2.29.0/)   |[0.13.5](https://github.com/hashicorp/terraform/releases/tag/v0.13.5)   | [0.16.1](https://docs.hashicorp.com/sentinel/changelog#0-16-1-october-21-2020) |
+| [v202102-1](./v202102-1.html) |  507              | [2.29.0](https://release-notes.replicated.com/release-notes/2.29.0/)   |[0.13.5](https://github.com/hashicorp/terraform/releases/tag/v0.13.5)   | [0.16.1](https://docs.hashicorp.com/sentinel/changelog#0-16-1-october-21-2020) |
+| [v202101-1](./v202101-1.html) |  504              | [2.29.0](https://release-notes.replicated.com/release-notes/2.29.0/)   |[0.13.5](https://github.com/hashicorp/terraform/releases/tag/v0.13.5)   | [0.16.1](https://docs.hashicorp.com/sentinel/changelog#0-16-1-october-21-2020) |
+
+\*  Release v202103-1 is a required release. All online upgrades will install this version before proceeding automatically, however airgap customers must upgrade to this version before proceeding to later releases.
```

Comment:
We should drop the specificity of `v202103-1` and just leave it at * denotes a required release just to be sure we don't miss updating this in the future.